test/controllers/oauth2_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+describe Oauth2Controller do
+  # it "does a thing" do
```

Comment:
Empty test.

Reply:
generator artifact.  I can update it in a follow up.